snap/local/with_config.sh

```diff
@@ -2,18 +2,12 @@
 
 set -e
 
-export CONFIG_FILE="${SNAP_COMMON}/zoo.cfg"
-if [ ! -f "${CONFIG_FILE}" ]; then
-	echo "custom configuration file ${CONFIG_FILE} does not exist."
-	echo "copying default config to ${CONFIG_FILE}"
-	cp $SNAP/conf/zoo_sample.cfg $CONFIG_FILE
-fi
-
-export PATH="${SNAP}/usr/lib/jvm/default-java/bin:${PATH}"
-export JAVA_HOME="${SNAP}/usr/lib/jvm/java-8-openjdk-amd64/jre"
+export JAVA_HOME="${SNAP_COMMON}/jvm/java-8-openjdk-amd64/jre"
```

Comment:
I think you could define most of these environment variables in your `snapcraft.yaml` file. 
Either globally (which seems to be your case), at the root:
```
environment:
  JAVA_HOME: ${SNAP_COMMON}/jvm/java-8-openjdk-amd64/jre
  ZOO_LOG_DIR: ${SNAP_COMMON}/logs
  ZOOCFGDIR: ${SNAP_COMMON}
```
(You can also define them per app, similar to above just under the `apps/app_name (init, daemon etc..)`

Reply:
This is helpful! Will have a look at adding that, thank you!